dwave/cloud/auth/flows.py

```diff
@@ -252,6 +253,14 @@ def _infer_auth_endpoint(leap_api_endpoint: str) -> str:
     def _infer_token_endpoint(leap_api_endpoint: str) -> str:
         return urljoin(leap_api_endpoint, '/leap/openid/token')
 
+    @staticmethod
+    def _infer_leap_success_uri(leap_api_endpoint: str) -> str:
+        return urljoin(leap_api_endpoint, '/leap/openid/success')
+
+    @staticmethod
+    def _infer_leap_error_uri(leap_api_endpoint: str) -> str:
+        return urljoin(leap_api_endpoint, '/leap/openid/error')
```

Comment:
I imagine that the answer is that these may eventually be used elsewhere, but why not just using `partial` in the `run_redirect_flow()` function body?

Reply:
I hope to replace some of these by reading them from server metadata endpoint, once a couple of issues are fixed on SAPI.